lib/cutting_edge/workers/badge.rb

```diff
@@ -1,20 +1,28 @@
 require File.expand_path('../../badge.rb', __FILE__)
 require File.expand_path('../helpers.rb', __FILE__)
 
+module CuttingEdge
+  BADGE_TEMPLATE = File.read(File.expand_path('../../templates/badge.svg.erb', __FILE__)) unless defined?(BADGE_TEMPLATE)
+  BADGE_BASE_WIDTH = 35 unless defined?(BADGE_BASE_WIDTH)
+  BADGE_CELL_WIDTH = 25 unless defined?(BADGE_CELL_WIDTH)
+  BADGE_COLORS = {outdated_major: 'red', outdated_minor: 'orange', outdated_patch: 'yellow', ok: '#4c1', unknown: '#555'} unless defined?(BADGE_COLORS)
+end
+
 class BadgeWorker < GenericWorker
+  STATUS_TYPES = [:ok, :outdated_patch, :outdated_minor, :outdated_major, :unknown]  # Order is significant
 
   def perform(identifier)
     log_info 'Running Worker!'
-    dependencies = get_from_store(identifier)
-    status = dependencies ? generate_status(dependencies[:outdated]) : :unknown
-    add_to_store("svg-#{identifier}", Badge.build_badge(status, dependencies[:outdated_total]))
-    GC.start
-  end
+    dependencies = get_from_store(identifier) # What if this fails?
```

Comment:
Address this

Reply:
Do you want to make the template conditional, and add a wider, red `rect` that says `error`? Or do you want a separate `error_badge.svg.erb` template?

Reply:
I think the error badge could even just be a static svg? Or would it need some dynamic element?

Reply:
Maybe not! In that case a static svg would do the trick. `CE | error` or `CE | Come back later` or `CE | Unknown problem` ?

Reply:
I like `error`. It's a brief but powerful call to action :)